react/react1/week3/todo-app-week3/src/App.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react';
+import Header from './components/Header';
+import Timer from './components/Timer';
+import TodoList from './components/TodoList';
+import Form from './components/Form';
+import './App.css';
+
+function App() {
+  const [input, setInput] = useState('');
+  const [todos, setTodos] = useState([]);
+  const [deadlineValue, setDeadlineValue] = useState(null);
+
+  const fetchData = async () => {
+    const data = await fetch(
+      'https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw'
+    ).then((res) => res.json());
+    setTodos(data);
+    console.log(data);
```

Comment:
remember to remove console.log and you could also add some error handling to catch exceptions here